Classes/DataProcessing/GridChildrenProcessor.php

```diff
@@ -31,9 +31,10 @@ protected function sortRecordsIntoMatrix(): array
                     if ($column['name']) {
                         $column['name'] = $GLOBALS['TSFE'] ? $GLOBALS['TSFE']->sL($column['name']) : $column['name'];
                     }
+                    $hasElements = isset($processedColumns[$column['colPos']]) && is_array($processedColumns[$column['colPos']]);
```

Comment:
@kitzberger Shouldn't we use `empty($processedColumns[$column['colPos']]) === false` (or !empty($processedColumns[$column['colPos']])) instead of `isset` since the above check does not necessarily say that it has elements?

Reply:
@schloram, when I tested it the other day the key wasn't set at all when having no (or only hidden) children in a column.

Reply:
```suggestion
                    $hasElements = !empty($processedColumns[$column['colPos']]) && is_array($processedColumns[$column['colPos']]);
```

Reply:
@kitzberger Could you please add the suggested changes? Then I'm fine to merge.

Reply:
@schloram, sure thing. Done.